galactory/utilities.py

```diff
@@ -20,6 +20,7 @@
 from flask.json.provider import DefaultJSONProvider
 from artifactory import ArtifactoryPath, ArtifactoryException
 from dohq_artifactory.auth import XJFrogArtApiAuth
+from dohq_artifactory.auth import XJFrogArtBearerAuth
```

Comment:
We can add this to the previous line, so that we have one line that looks like

```python
from dohq_artifactory.auth import XJFrogArtApiAuth, XJFrogArtBearerAuth
```